### ISSUE TITLE
Wiring diagram

### ISSUE THREAD

**Issue by That-Dude** [epiclabs-io/esphome-modbus-server#3]
First of all, thanks for your work, im excited to try it out. 

How are you connecting the esp32 to modbus? A Max485 module or something similar?

A simple wiring diagram would be much appreciated.

**Comment by wspeth** [comment]
> First of all, thanks for your work, im excited to try it out.
> 
> How are you connecting the esp32 to modbus? A Max485 module or something similar?
> 
> A simple wiring diagram would be much appreciated.

Curious whether the same hardware was employes like 
https://esphome.io/components/modbus_controller.html

**Comment by jpeletier** [comment]
Yes, the wiring is the same as in https://esphome.io/components/modbus_controller.html
